sdk/js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pythnetwork/express-relay-js",
-  "version": "0.14.3",
+  "version": "0.15.0",
```

Comment:
I'm not sure about best practice here
Every time limo update, are we gonna increase the version like this? :-?

Reply:
I did this because of the introduction of new bid status and not because of limo